src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTestsLang.java

```diff
@@ -13,16 +13,21 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep2;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep3;
 
+
+import static com.wikia.webdriver.common.core.Assertion.assertEquals;
+
 @Test(groups = {"CNW_lang"})
-public class CreateWikiTests_lang extends NewTestTemplate {
+public class CreateWikiTestsLang extends NewTestTemplate {
 
   @Test(dataProviderClass = CreateNewWikiDataProvider.class, dataProvider = "getLangs",
       groups = {"CreateNewWiki_lang_001", "CNW_lang_first"})
   @Execute(asUser = User.USER_CNW)
-  public void CreateNewWiki_lang_TC001(String lang) {
+  public void createNewWikiLangTC001(String lang) {
     WikiBasePageObject base = new WikiBasePageObject();
     CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
     cnw1.selectLanguage(lang);
+    String expectedDomainPrefix = lang.equals("en") ? "" : String.format("%s.", lang);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "en" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)